README.md

```diff
@@ -105,15 +108,22 @@ Alternatively a configuration file can be used instead of argument flags.
 With the following configuration file `config.yaml`:
 
 ```
-url: localhost
-# grafana
-grafana-service-token: glsa_12345
-grafana-dashboard: tests/dashboard.json
-shared-data-path: $SNAP_COMMON/rob-cos-shared-data setup
+url: http://cos-server/cos-robotics-model-cos-registration-server/
+uid: my-robot-uid
+grafana-dashboards: path/to_grafana_dashboards/
+foxglove-studio-dashboards: path/to_foxglove_studio_dashboards/
+loki-alert-rule-files: path/to_loki_alert_rule_files
+prometheus-alert-rule-files: path/to_prometheus_alert_rule_files
+setup
+device-grafana-dashboards: [dashboard-1, dashboard-2]
+device-foxglove-dashboards: [dashboard-3, dashboard-4]
+device-loki-alert-rule-files: None
+device-prometheus-alert-rule-files: [alert-rule-file-1]
+
 ```
 Then we can call `cos-registration-agent` with:
 ```
-cos-registration-agent --config ./config.yaml setup
+cos-registration-agent --config ./config.yaml
```

Comment:
and then here cos-reg-agent --config setup? it has changed?

Reply:
yes since the setup is in the `config.yaml`